alacritty/src/string.rs

```diff
@@ -50,8 +50,9 @@ impl<'a> StrShortener<'a> {
         }
 
         if direction == ShortenDirection::Right {
+            let skip_chars = 0; // suppress clippy warning about `skip(0)` in Rust 1.73+
             return Self {
-                chars: text.chars().skip(0),
+                chars: text.chars().skip(skip_chars),
```

Comment:
This is pretty garbage, why not just add a clippy exception?

Reply:
That warning is showing with clippy from Rust 1.73.0 (the current stable). We are supporting Rust 1.70.0 and newer. Clippy prior to 1.73.0 doesn't understand `#[allow(clippy::iter_skip_zero)]`. So I'm also allowing `unknown_lint` and adding a comment why it's needed.